main.js

```diff
@@ -4,7 +4,7 @@ import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls';
 
 // Setup
 
-const scene = new THREE.Scene();
+const scene = new THREE.Scene(); // CHQ: adding a comment to complete my agenda of having the original code deployed in a branch, for testing
```

Comment:
By making a new branch for this simple change, I was able to make a pull request for the new branch to main. In essence, making a new branch for this comment achieved the goal of having the main branch treat a pull request from this branch as an update, when it is in fact, a reversion. 

Additionally, the deployment environment tested this reverted version of the repository. The bugs experienced in the current version are the same bugs present in the original, reverted version. 

Good news: the issue is not with the changes that I have made
Bad news: I must continue searching for how to fix the bugs